src/main/java/org/fundacion/jala/converter/core/facade/DownloadLinkFacade.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Zacarias Huanca Alconz
+ */
+package org.fundacion.jala.converter.core.facade;
+
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+/**
+ * Class to do link to download.
+ */
+public class DownloadLinkFacade {
+    private static String baseUrl = ServletUriComponentsBuilder.fromCurrentContextPath().build().toUriString();
+
+    private DownloadLinkFacade() {
+    }
+
+    /**
+     *  Generator link to download to convert.
+     *
+     * @param outputFilename is path of file which are converted.
+     * @return path which permit download the result process.
```

Comment:
Maybe you can add the type of variable that returns, e.g. @return a String path which.......

Reply:
I forgot it, thanks